bundles/cif-connector-graphql/src/main/java/com/adobe/cq/commerce/postprocessor/MultiFieldDropTargetPostProcessor.java

```diff
@@ -103,6 +106,21 @@ public void process(SlingHttpServletRequest request, List<Modification> modifica
                     String propertyValue = requestParameter.getString();
                     Resource resource = request.getResource();
 
+                    // If it's actually a drag and drop, the POST contains a Sling './@CopyFrom' parameter
+                    // and the modifications map will already contain the COPY operation done by the D'n'D
+                    if (requestParameterMap.containsKey(SLING_COPY_FROM)) {
```

Comment:
Is it "save" in the way that other component changes like from dialog don't have a `./@CopyFrom` parameter?

Reply:
Keep in mind that the entire post processor is already "protected" by the fact that it only handles requests that contain a request parameter that starts with `./multiDropTarget->` so even if this is used elsewhere, it won't be affected.

Reply:
Ah yes correct. 👍